src/main/java/frc/robot/subsystems/Swerve.java

```diff
@@ -155,22 +188,21 @@ public void logPositions() {
                 frontLeft.getState(), frontRight.getState(), rearLeft.getState(), rearRight.getState()
         };
 
-        ChassisSpeeds speeds = DriveConstants.DRIVE_KINEMATICS.toChassisSpeeds(RobotContainer.swerveMeasuredStates);
+        currentChassisSpeeds = DriveConstants.DRIVE_KINEMATICS.toChassisSpeeds(RobotContainer.swerveMeasuredStates);
 
         if (FieldConstants.IS_SIMULATION) {
             resetOdometry(
                     currentPose.exp(
                             new Twist2d(
                                     0, 0,
-                                    speeds.omegaRadiansPerSecond * .02)));
+                                    currentChassisSpeeds.omegaRadiansPerSecond * .02)));
```

Comment:
ive never seen more indents!

Reply:
Bro I need to find that score setting ;( something is wrong 😞